src/sdk/refundPegout.ts

```diff
@@ -1,12 +1,25 @@
 import { assertTruthy, throwErrorIfFailedTx, validateRequiredFields } from '@rsksmart/bridges-core-sdk'
 import { type PegoutQuote, pegoutQuoteDetailRequiredFields, pegoutQuoteRequiredFields } from '../api'
-import { type LiquidityBridgeContract } from '../blockchain/lbc'
+import { type FlyoverSDKContext } from '../utils/interfaces'
+import { isPegoutRefundable } from './isPegoutRefundable'
+import { FlyoverError } from '../client/httpClient'
 
-export async function refundPegout (quote: PegoutQuote, lbc: LiquidityBridgeContract): Promise<string> {
+export async function refundPegout (quote: PegoutQuote, context: FlyoverSDKContext): Promise<string> {
   validateRequiredFields(quote, ...pegoutQuoteRequiredFields)
   validateRequiredFields(quote.quote, ...pegoutQuoteDetailRequiredFields)
```

Comment:
DRY?  can be wrapped into something like validatePegoutQuote ?

Reply:
This is not a repetition. This could be wrapped but it will be a function with just two sentences and will grow the size of the code with not much functionality of clarity. The functionality to validate the required fields of an object is already encapsulated in `validateRequiredFields` and I think this decision of making two separated calls is correct. Let me know your opinion on that.